### ISSUE TITLE
Can't rename file saves in Lunux: Itch.io sandboxing

### ISSUE THREAD

**Issue by SojanJanso** [armory3d/armorpaint#142]
ArmorPaint bfed2ba
Linux (Ubuntu Gnome)

When saving any project in armory, the name set in the text input is "--file-filter=*.arm", and no text can be changed in this field.

**Comment by luboslenco** [comment]
Was this working in the past for you? I get the same "--file-filter=" text on manjaro but can edit it normally. I will need to run more linux distros to make it more reliable. 😢 

On Linux armorpaint uses zenity from https://github.com/mlabbe/nativefiledialog.

**Comment by SojanJanso** [comment]
I honestly don't know, since save was rather unreliable, so I never used it in the past :)
![Peek 2019-06-19 15-17](https://user-images.githubusercontent.com/26679320/59793827-87603880-92a5-11e9-94b8-091e55d2bdd3.gif)

**Comment by SojanJanso** [comment]
Also, I'm not familiar with manjaro, but does it also use the [GtkFileChooserDialog](https://developer.gnome.org/gtk3/stable/GtkFileChooserDialog.html)

**Comment by SojanJanso** [comment]
Hmm, after a reboot and building 31fdedb, it works. The only thing is, I went back to the itch version (bfed2ba) and it wouldn't work. When I tried launching the file on it's own (in the .UserName/.config/itch/app) it worked fine. Turns out, when launching from the itch.io app, if the itch.io "sandbox" option is enabled, that is what kept me from modifying the name. So your app is all good on its own.
